.github/workflows/update_xdsl_pyodide_build.py

```diff
@@ -3,10 +3,11 @@
 # This script updates the meta.yaml file used by Pyodide to bundle and use xDSL
 # Takes the .yaml file and the xDSL directory as arguments
 
-import yaml  # type: ignore
-import sys
-import os
 import hashlib
+import os
+import sys
+
+import yaml  # type: ignore
```

Comment:
why type ignore? Can you replace with specific ignore as opposed to catch-all, please?

Reply:
This change is not part of the PR, you see the diff from merging main into this branch